snapcraft/internal/lxd.py

```diff
@@ -38,6 +40,11 @@
 
 class Containerbuild:
 
+    def _get_server_info(self):
```

Comment:
Another nit, can you move this method after __init__?
I love some of the suggestions from the [clean code](https://www.safaribooksonline.com/library/view/clean-code/9780136083238/) book. One of those is vertical organization, you put first the caller method, and after the called method. This leaves the functions close to were they were called first, but the nicest thing for me is that you can read a source code file from top to bottom. In here if you find _get_server_info you have no context about what it does. If you instead find __init__ first, you will see the context where _get_server_info is called and then you can decide if the internals of that method are relevant to what you are looking for, or you can just skip it.